include/boost/numeric/odeint/stepper/adaptive_adams_bashforth.hpp

```diff
@@ -0,0 +1,152 @@
+#ifndef STEPPER_ADAMS_BASHFORTH_HPP_INCLUDED
+#define STEPPER_ADAMS_BASHFORTH_HPP_INCLUDED
+
+#include <boost/numeric/odeint/stepper/detail/polynomial.hpp>
+#include <boost/numeric/odeint/stepper/detail/adaptive_adams_coefficients.hpp>
+
+#include <boost/numeric/odeint/stepper/base/algebra_stepper_base.hpp>
+#include <boost/numeric/odeint/algebra/algebra_dispatcher.hpp>
+#include <boost/numeric/odeint/algebra/operations_dispatcher.hpp>
+
+#include <boost/numeric/odeint/stepper/stepper_categories.hpp>
+
+#include <boost/numeric/odeint/util/copy.hpp>
+#include <boost/numeric/odeint/util/bind.hpp>
+
+#include <boost/numeric/odeint/util/state_wrapper.hpp>
+#include <boost/numeric/odeint/util/is_resizeable.hpp>
+#include <boost/numeric/odeint/util/resizer.hpp>
+
+#include <iostream>
+
+namespace boost {
+namespace numeric {
+namespace odeint {
+
+template<
+size_t Steps,
+class State,
+class Value = double,
+class Deriv = State,
+class Time = Value,
+class Algebra = typename algebra_dispatcher< State >::algebra_type,
+class Operations = typename operations_dispatcher< State >::operations_type ,
+class Resizer = initially_resizer
+>
+class adaptive_adams_bashforth: public algebra_stepper_base< Algebra , Operations >
+{
+	public:
+		static const size_t steps = Steps;
+		typedef unsigned short order_type;
+		static const order_type order_value = steps;
```

Comment:
Is this really true? If I understand the implementation below correctly it calls the rhs function exactly once? I find it hard to believe this can result in a higher order.